examples/worlds/lift_drag.sdf

```diff
@@ -2,7 +2,7 @@
 <!--
   Gazebo lift-drag system demo
 
-  This file was adapted from osrf/gazebo/worlds/single_rotor_demo.world
+  This file was adapted from osrf/sim/worlds/single_rotor_demo.world
```

Comment:
Oh this is actually `gazebo`

https://github.com/osrf/gazebo/

Reply:
Reverted!